### ISSUE TITLE
Marking a locked entry as ham unlocks the entry

### ISSUE THREAD

**Issue by auge8472** [ilosuna/mylittleforum#615]
I marked very old entries, which was locked, as ham in the project forum and became aware that all these ancient, locked threads got unlocked after ham-classification of their entries. Outcome of testing the issue is, that every entry, that is part of a locked thread, is unlocked after it is classified as ham. Other entries in the same thread stay locked until they are also classified as ham.

Spam entries on the other hand will be deleted normally, so their lock-status is mostly not relevant. Entries that was detected as spam by the script itself, get locked automatically. I will check the situation for entries, that are *manually* marked as spam and was locked before.

**Comment by loesler** [comment]
Yes, this is a side effect because a (wrong) classified SPAM posting is automatically locked.

**Comment by auge8472** [comment]
Thank you for the hint.

**Comment by auge8472** [comment]
After a few weeks of working with the changed code in the project forum, I have to state that there are edge cases, that still lead to unlocking entries when classifying them as ham. My observation is, that only entries of not registered users seems to be affected. A further constraint seems to be, that the thread opening entry has to be an entry of a not registered user.

Currently I can't see what causes this behaviour. So I have to reopen the issue.
